src/bin/ion/commands/generate/model.rs

```diff
@@ -448,6 +455,58 @@ pub struct FieldReference(
     pub(crate) FieldPresence,
 );
 
+/// Represents an enum type
+/// e.g. Given below ISL,
+/// ```
+/// type::{
+///   name: enum_type,
+///   valid_values: [foo, bar, baz]
+/// }
+/// ```
+/// Corresponding generated code in Rust would look like following:
+/// ```
+/// enum EnumType {
+///    Foo,
+///    Bar,
+///    Baz
+/// }
+/// ```
+#[allow(dead_code)]
+#[derive(Debug, Clone, Builder, PartialEq, Serialize)]
+#[builder(setter(into))]
+pub struct Enum {
+    // Represents the fully qualified name for this data model
+    pub(crate) name: FullyQualifiedTypeName,
+    // Represents enum variants
+    variants: HashSet<EnumVariant>,
+    // Represents doc comment for the generated code
+    // If the doc comment is provided for this scalar type then this is `Some(doc_comment)`, other it is None.
```

Comment:
Contains a typo ("scalar" rather than "enum"), but it's stating the obvious so just remove it, I think.
```suggestion
```